CoreFoundation/Locale.subproj/CFDateFormatter.c

```diff
@@ -1517,7 +1524,7 @@ static UDate __CFDateFormatterCorrectTimeToARangeAroundCurrentDate(UCalendar *ca
         }
     } else {
         if (period < INT_MAX && futureMax > period) {
-            futureRange.location = 1;
+            futureRange.location = 1,
```

Comment:
Oh. This must've reverted a fix inadvertently.

Reply:
Just for the record, in case it's somehow relevant in the future, the comma operator has a lower precedence than the assignment operator so this code does work as expected with the comma, although it's definitely weird. It's equivalent to `(futureRange.location = 1), (futureRange.length = futureMax - period);`. This very occasionally has a practical use in `for` loops, where you can use it to do multiple assignments in the increment section of the `for()`, but mostly this tidbit is only useful for making strange code.